api/InstallEvent.json

```diff
@@ -30,8 +31,8 @@
         },
         "status": {
           "experimental": false,
-          "standard_track": false,
-          "deprecated": true
+          "standard_track": true,
+          "deprecated": false
         }
       },
       "InstallEvent": {
```

Comment:
the `InstallEvent()` constructor can also mark as standard now, too

Reply:
Where in the spec do you see the constructor? https://w3c.github.io/ServiceWorker/#installevent

Reply:
well, my mistake...